webapp/migrations/Version20250620082406.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20250620082406 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Change comments to reflect entities';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE judging CHANGE max_runtime_for_verdict max_runtime_for_verdict NUMERIC(32, 9) UNSIGNED DEFAULT NULL COMMENT \'The maximum runtime for all runs that resulted in the verdict\'');
```

Comment:
It took me a while to spot the difference.. If we're already doing this and touching the database, should it also be `max_run_time_for_verdict` as now the comment and column name imply to different things. Because now we imply the largest environment with the column and the longest run for the judging. I think it's important to make sure the columns imply the right thing in case someone ever wants to rate on the smallest resulting binary for example or more serious, if we change this let's be consistent.

Reply:
The comment changed from `run time` to `runtime`, to be in sync with what is in the entity file.